### DISCUSSION TITLE
Block forged sender using multimap?

### DISCUSSION THREAD

**Discussion by dhana-exe** [rspamd/rspamd#4427] [Q&A]
hello guys
how do i block people when they mail is forged?

example
[[010901869e31528a-46aa3856-4ded-4278-a96b-68f658035981-000000@ap-south-1.amazonses.com](mailto:010901869e31528a-46aa3856-4ded-4278-a96b-68f658035981-000000@ap-south-1.amazonses.com)] [info@aiplex.com](mailto:info@aiplex.com)

this mail come from aiplex, i already blocked them via multimap and works, but seems that "amazonses" is the sender if they are forged
so the question how can i block them without blocking other's amazon ses email?

i just want to get rid of this aiplex... they keep sending email everyday

**Comment by fatalbanana** [comment]
If it is safe to require SPF pass for amazonses.com (I suppose it is not, YMMV), you could use whitelist module to apply scoring in case message would fall foul of this policy (or do something with this symbol later if you must be sure to reject it): https://www.rspamd.com/doc/modules/whitelist.html

It should be generally safer to require DMARC but I don't see that aiplex.com has a DMARC policy.

**Reply by dhana-exe** [reply]
any other solution to block it?
![image](https://user-images.githubusercontent.com/29838157/227286487-873d8b78-5bef-4101-b948-eb335ef5cb53.png)
on past i already blocked all amazonses.com but it seems that i doing wrong things....
alot of important mail come from them....

and this shitty aiplex seems doesn't even care about my email reply
they just flooded my mail with spam email..... even i already answered they re-send it on next day
